vscode/src/client/analyzerClient.ts

```diff
@@ -130,29 +172,29 @@ export class AnalyzerClient {
         args: getConfigKaiProviderArgs(),
       },
       file_log_level: getConfigLogLevel(),
-      demo_mode: demoMode,
+      demo_mode: this.isDemoMode(),
       cache_dir: "",
 
-      analyzer_lsp_java_bundle_path: path.join(
-        this.extContext!.extensionPath,
-        "assets/bin/jdtls/java-analyzer-bundle/java-analyzer-bundle.core/target/java-analyzer-bundle.core-1.0.0-SNAPSHOT.jar",
-      ),
-      analyzer_lsp_lsp_path: path.join(
-        this.extContext!.extensionPath,
-        "assets",
-        "bin",
-        "jdtls",
-        "bin",
-        "jdtls",
-      ),
+      // Analyzer and jdt.ls parameters
       analyzer_lsp_rpc_path: this.getAnalyzerPath(),
-      analyzer_lsp_rules_path: this.getRules(),
-      analyzer_lsp_dep_labels_path: path.join(
-        this.extContext!.extensionPath,
-        "assets/bin/jdtls/java-analyzer-bundle/maven.default.index",
-      ),
+      analyzer_lsp_lsp_path: this.assetPaths.jdtlsBin,
+
+      // jdt.ls bundles (comma separated list of paths)
+      analyzer_lsp_java_bundle_path: this.assetPaths.jdtlsBundleJars.join(","),
+
+      // depOpenSourceLabelsFile
+      analyzer_lsp_dep_labels_path: this.assetPaths.openSourceLabelsFile,
+
+      // TODO: Do we need to include `fernFlowerPath` to support the java decompiler?
+      // analyzer_lsp_fernflower: this.assetPaths.fernFlowerPath,
```

Comment:
Absolutely keep the TODO, but looking at the [parameters to initialize in kai](https://github.com/konveyor/kai/blob/87aa4c1455b66d5e7997d0e1bf06eab228791ec4/kai/rpc_server/server.py#L58), they don't currently take fernflower.

Reply:
I traced through the various parameters yesterday mostly to see if the bundle jar paths could be joined.  Answer: yes, the param is spit by "," in the golang code.  That's where I saw fernflower references.  And when I chatted with Parnav about what assets to keep in the repo (for now), I asked about fernflower.  He said yes but not with authority.  So it is there should it be needed.  I also thought it what a jdtls bundle for quite some time (it isn't).